scripts/config/templates/nginx.conf.erb

```diff
@@ -163,6 +163,12 @@ http {
     proxy_set_header X-Forwarded-Proto  $scheme;
   }
 
+  # Redirect our locations pages to the new url structure
+  location ~ "(*UTF8)^\/locations\/[A-z][A-z]\/[A-z\-\/]*$" {
+    mruby_set $locations_path /app/bin/config/lib/ngx_mruby/locations_state_path.rb cache;
+    return 301 https://$host/locations/$locations_path$is_args$args;
```

Comment:
If the only result of all location matches is 301.... will we always have a 301 redirect on the first request?

Reply:
This will only match the `/locations` urls with `state` being 2 letters. All other `locations` will return 200

Reply:
https://user-images.githubusercontent.com/55200415/156630418-73292a30-f11f-465d-98e3-796eb0443556.mov